packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -25,7 +25,9 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
           [`.${IsFocusVisibleClassName} &:focus`]: {
             // Can't use getFocusStyle because it doesn't support wrapping links
             // https://github.com/OfficeDev/office-ui-fabric-react/issues/4883#issuecomment-406743543
-            outline: `1px solid ${theme.palette.neutralSecondary}`
+            // A box-shadow allows the focus rect to wrap links that span multiple lines
+            // and helps the focus rect avoid getting clipped.
+            boxShadow: '0 0 0 1px black inset'
```

Comment:
We should still be using `neutralSecondary` from the theme here, right?

Reply:
yes

Reply:
changed it back to nuetralSecondary